rct229/rules/section6/section6rule9.py

```diff
@@ -0,0 +1,135 @@
+                            rmrs_used=UserBaselineProposedVals(False, True, True)
+                        )
+
+                    def get_calc_vals(self, context, data=None):
+                        space_p = context.proposed
+                        space_b = context.baseline
+                        avg_space_height = data["avg_space_height"]
+                        schedules_b = data["schedules_b"]
+                        schedules_p = data["schedules_p"]
+                        building_open_schedule_p = data["building_open_schedule_p"]
+
+                        normalized_schedule_b = normalize_interior_lighting_schedules(space_b, avg_space_height, schedules_b)
+                        normalized_schedule_p = normalize_interior_lighting_schedules(space_p, avg_space_height, schedules_p)
+                        schedule_comparison_result_dictionary = compare_schedules(normalized_schedule_p, normalized_schedule_b,
+                                                                       building_open_schedule_p, 1.0)
+                        daylight_control = functools.reduce(lambda daylight_control_a, daylight_control_b: daylight_control_a | daylight_control_b, find_all("interior_lighting[*].are_schedules_used_for_modeling_daylighting_control", space_p))
+
+                        return{
+                            "daylight_control": daylight_control,
+                            "total_hours_compared": schedule_comparison_result_dictionary["total_hours_compared"],
+                            "total_hours_matched": schedule_comparison_result_dictionary["total_hours_matched"],
+                            "eflh_difference": schedule_comparison_result_dictionary["eflh_difference"]
+                        }
+
+                    def manual_check_required(self, context, calc_vals=None, data=None):
+                        daylight_control = calc_vals["daylight_control"]
+                        return daylight_control
+
+                    def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+                        elfh_differeces = calc_vals["eflh_difference"]
+                        return f"Space models at least one daylight control using schedule. Verify if other programmable lighting control is modeled correctly using schedule. Lighting schedule EFLH in P-RMD is {elfh_differeces} of that in B-RMD"
+
+                    def rule_check(self, context, calc_vals=None, data=None):
+                        total_hours_compared = calc_vals["total_hours_compared"]
+                        total_hours_matched = calc_vals["total_hours_matched"]
+                        return total_hours_matched == total_hours_compared
+
+                    def get_fail_msg(self, context, calc_vals=None, data=None):
+                        eflh_difference = calc_vals["eflh_difference"]
+                        return f"Space lighting schedule EFLH in P-RMD is {eflh_difference} of that in B-RMD"
```

Comment:
period(.) is missing here. Could you also check if we need to update the ```B-RMR``` to ```B-RMD``` in the RDS?